src/nvim/main.c

```diff
@@ -1367,7 +1367,8 @@ static void load_plugins(void)
 {
   if (p_lpl) {
     char_u *rtp_copy = NULL;
-    char_u *const plugin_pattern = (char_u *)"plugin/**/*.vim";  // NOLINT
+    char_u *const plugin_pattern_vim = (char_u *)"plugin/**/*.vim";  // NOLINT
```

Comment:
Is there no way to do `*.(vim|lua)` or other -- I don't remember which matching is used here :laughing:

Reply:
No . It doesn't work I've tried that